0.3.0: iOS build error

As a result of commit https://github.com/smooch/react-native-smooch/commit/ebc802888071479b2262ebca0dc095754c42b7f5 and the corresponding release 0.3.0 of this library, I now see the following problem when I try to run iOS simulator build:

```
[my_private_path]/node_modules/react-native-smooch/ios/RCTSmooch/RCTSmooch.m:46:19: error: too many arguments provided to function-like macro invocation
                  resolver:(RCTPromiseResolveBlock)resolve
                  ^
In file included from [my_private_path]/node_modules/react-native-smooch/ios/RCTSmooch/RCTSmooch.m:1:
In file included from [my_private_path]/node_modules/react-native-smooch/ios/RCTSmooch/RCTSmooch.h:4:
../../react-native/React/Base/RCTBridgeModule.h:144:9: note: macro 'RCT_EXPORT_METHOD' defined here
#define RCT_EXPORT_METHOD(method) \
        ^
[my_private_path]/node_modules/react-native-smooch/ios/RCTSmooch/RCTSmooch.m:45:1: warning: type specifier missing, defaults to 'int' [-Wimplicit-int]
RCT_EXPORT_METHOD(getUnreadCount,
^
[my_private_path]/node_modules/react-native-smooch/ios/RCTSmooch/RCTSmooch.m:45:18: error: expected ';' after top level declarator
RCT_EXPORT_METHOD(getUnreadCount,
                 ^
                 ;
1 warning and 2 errors generated.



** BUILD FAILED **
```

Unfortunately, I don't know the first thing about Objective-C and haven't a clue as to how to fix this.  @Leeds-eBooks are you seeing this?

--- a/ios/RCTSmooch/RCTSmooch.m
+++ b/ios/RCTSmooch/RCTSmooch.m
@@ -42,13 +42,13 @@ RCT_EXPORT_METHOD(track:(NSString*)eventName) {
   [Smooch track:eventName];
 };
 
-RCT_EXPORT_METHOD(getUnreadCount,
+RCT_REMAP_METHOD(getUnreadCount,
                   resolver:(RCTPromiseResolveBlock)resolve
                   rejecter:(RCTPromiseRejectBlock)reject) {
   NSLog(@"Smooch getUnreadCount");
 
-  NSUInteger *unreadCount = [Smooch conversation].unreadCount;
-  resolve(unreadCount);
+  long unreadCount = [Smooch conversation].unreadCount;
+  resolve(@(unreadCount));
 };
 
 RCT_EXPORT_METHOD(setFirstName:(NSString*)firstName) {

